src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/Licence.kt

```diff
@@ -37,24 +38,29 @@ sealed interface Licence {
   val id: Long
```

Comment:
if we're exposing licence id on events we should probably expose it here too

Reply:
should probably expose kind too

Reply:
we are already exposing kind.